src/main/resources/hudson/plugins/ec2/SlaveTemplate/help-connectBySSHProcess.html

```diff
@@ -0,0 +1,34 @@
+<!--
+The MIT License
+
+Copyright (c) 2004-, Kohsuke Kawaguchi, Sun Microsystems, Inc., and a number of other of contributors
```

Comment:
@KostyaSha Hmmm? I don't have any created-by/author attributions on the file. The _**Copyright**_ decl is the same as the other files in there. You saying it should be different? I wouldn't think so.